backend/exports/graphql/schema.graphql

```diff
@@ -10,6 +10,102 @@ type Comment {
   SomeText: String
 }
 
+type CommentAggregator {
```

Comment:
Not sure why these were added? I didn't add these manually? I suppose during the upgrade to beta.20?